src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,15 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. You should use private access modifier for fields and provide public getters and setters for accessing them.